config/routes.rb

```diff
@@ -13,6 +13,9 @@
   post 'signup' => 'user_registration#create'
   post 'logout' => 'user_sessions#destroy', :as => :logout
 
+  delete 'decks' => 'user_profile#reset_current_deck', :as => :reset_current_deck
+  put 'decks' => 'user_profile#set_current_deck', :as => :set_current_deck
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.